src/Camille.RiotGames/gen/configspec.json

```diff
@@ -3,6 +3,21 @@
     "type": "string",
     "desc": "Riot Games API key."
   },
+  "apiUrl": {
+    "type": "string",
+    "desc": "URL to call for the Riot Games API. Include {0} for the region, if you want it in the URL - Not necessary for RegionConfig.inUrlAsRegionQueryParameter.",
+    "val": "\"{0}.api.riotgames.com\""
+  },
+  "apiCallRegionConfig": {
+    "type": "RegionConfig",
+    "desc": "How the API call should include the region.",
+    "val": "RegionConfig.InUrlAsSubdomain"
+  },
+  "regionHeaderKey": {
+    "type": "string",
+    "desc": "The header key to use for the region. Only used if RegionConfig is set to inHeader.",
+    "val": "\"Riot-Region\""
+  },
```

Comment:
Could you make this work with `InUrlAsRegionQueryParameter` too and just call it `ReigonKey`? 

"The query or header name to use for the region. Only used if RegionConfig is set to InHeader or InQuery."